ReSharper.FSharp/src/FSharp/FSharp.Common/src/Shim/AssemblyReader/ProjectFcsModuleReader.fs

```diff
@@ -1260,13 +1264,23 @@ type ProjectFcsModuleReader(psiModule: IPsiModule, cache: FcsModuleReaderCommonC
                 let properties = mkILPropertiesLazy (InterruptibleLazy(fun _ -> getOrCreateProperties membersTable clrTypeName))
                 let events = mkILEventsLazy (InterruptibleLazy(fun _ -> getOrCreateEvents membersTable clrTypeName))
 
-                let customAttrs = mkTypeDefCustomAttrs typeElement |> storeILCustomAttrs
+                // TODO: avoid hasExtensions in closure?
+                let hasExtensions = hasExtensions typeElement
```

Comment:
This looks cheap enough to calculate it again when creating the attributes. Let's not capture it indeed?